### PR TITLE
TEST: Increase timeout testFollowIndexAndCloseNode

### PR DESCRIPTION
**Pull request by dnhatn**
This test fails several times due to timeout when asserting the number
of docs on the following and leading indices. This change reduces
the number of docs to verify and increases the timeout.

### CODE REVIEW

**Comment by dnhatn** [comment]
CI: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+ccr+feature-branch-periodic/1169/console

File: x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/ShardChangesIT.java [hunk 1]
```diff
@@ -338,26 +337,12 @@ public void testFollowIndexAndCloseNode() throws Exception {
...
-                    throw new UncheckedIOException(e);
-                }
-                return true;
-            } else {
-                return false;
-            }
-        }, 30, TimeUnit.SECONDS);
-
+        atLeastDocsIndexed("index2", minNumDocsReplicated);
+        internalCluster().stopRandomNonMasterNode();
```
Comment (martijnvg) [inline x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/ShardChangesIT.java]: 👍 much cleaner. There was no need to stop a node inside the callback.

**Review by martijnvg — APPROVED** [review]
LGTM

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by dnhatn** [comment]
Thanks @martijnvg and @jasontedor.

**Pull request merged**